### PR TITLE
Fix plugin admin UI overlapping

### LINKED ISSUE #12229: Plugin Admin UI glitch
### Description of the Issue

There are few controls on plugin admin which overlaps.

### Steps to Reproduce the Issue

1. Compile current notepad++ code
2. Open plugin admin and see all the controls

### Expected Behavior

There should no overlap on UI controls.
image

### Actual Behavior

See the highlighted 
image

### PR DESCRIPTION
**Pull request by SinghRajenM**
Fixes #12229

### CODE REVIEW

File: PowerEditor/src/WinControls/PluginsAdmin/pluginsAdmin.rc [hunk 1]
```diff
@@ -35,8 +35,8 @@ BEGIN
 	PUSHBUTTON      "Update", IDC_PLUGINADM_UPDATE,432,35,57,14
 	PUSHBUTTON      "Remove", IDC_PLUGINADM_REMOVE,432,35,57,14
 	EDITTEXT        IDC_PLUGINADM_EDIT,30,160,220,80,ES_MULTILINE | ES_AUTOVSCROLL | ES_READONLY | WS_BORDER | WS_VSCROLL
-	RTEXT           "Plugin list version: ", IDC_PLUGINLIST_VERSION_STATIC,350,230,120,14
-	LTEXT           "", IDC_PLUGINLIST_VERSIONNUMBER_STATIC,470,230,30,14
-	RTEXT           "Plugin list repository",IDC_PLUGINLIST_ADDR,368,245,120,14
+	RTEXT           "Plugin list version: ", IDC_PLUGINLIST_VERSION_STATIC,350,242,120,14
+	LTEXT           "", IDC_PLUGINLIST_VERSIONNUMBER_STATIC,470,242,30,14
+	RTEXT           "Plugin list repository",IDC_PLUGINLIST_ADDR,368,257,120,14
```
Comment (donho) [inline PowerEditor/src/WinControls/PluginsAdmin/pluginsAdmin.rc:40]: Why not?
But I would do the following:
image

**Review by donho — CHANGES_REQUESTED** [review]
Reduce the height of tab border.
Move 2 new added lines down to align with tab border.

File: PowerEditor/src/WinControls/PluginsAdmin/pluginsAdmin.cpp [hunk 2]
```diff
@@ -146,7 +146,7 @@ void PluginsAdminDlg::create(int dialogID, bool isRTL, bool msgDestParent)
 	_tab.insertAtEnd(installed);
 	_tab.insertAtEnd(incompatible);
 
-	rect.bottom -= dpiManager.scaleX(100);
+	rect.bottom -= dpiManager.scaleY(105);
```
Comment (SinghRajenM) [inline PowerEditor/src/WinControls/PluginsAdmin/pluginsAdmin.cpp:149]: This is to reduce the overall height of tab control.

File: PowerEditor/src/WinControls/PluginsAdmin/pluginsAdmin.cpp [hunk 3]
```diff
@@ -204,7 +204,7 @@ void PluginsAdminDlg::create(int dialogID, bool isRTL, bool msgDestParent)
 	RECT listRect = rect;
 	RECT descRect = rect;
 
-	long descHeight = rect.bottom / 3 - marge * 2;
+	long descHeight = rect.bottom / 3 - marge;
```
Comment (SinghRajenM) [inline PowerEditor/src/WinControls/PluginsAdmin/pluginsAdmin.cpp:207]: This is to reduce the margin between tab border and description list control

**Review by SinghRajenM — COMMENTED** [review]
Addressed review comment

File: PowerEditor/src/WinControls/PluginsAdmin/pluginsAdmin.rc (hunk 1 above)
Comment (SinghRajenM) [reply PowerEditor/src/WinControls/PluginsAdmin/pluginsAdmin.rc]: @donho Done!

**Review by donho — APPROVED** [review]

**Pull request closed without merging**